config-helpers.go

```diff
@@ -34,9 +35,9 @@ func envConfigIntWithDefault(key string, defValue int) int {
 // Parsing the "bucket/path:project/key,bucket2:key2"
 func bucketKeyMappings(bucketKeyMapString string) map[string]string {
 
-	fmt.Println(bucketKeyMapString)
+	log.Debug(bucketKeyMapString)
 	if bucketKeyMapString==""{
-		fmt.Println("No Bucket Key Mapping given , so using the default key for encryption and decryption")
+		log.Debug("No Bucket Key Mapping given , so using the default key for encryption and decryption")
```

Comment:
KeyMap cann't be empty.  There is no default key.  User has to provide the key(s). If it's empty, it's a fatal error and the proxy should exit as the purpose of the proxy sidecar is to enforce encryption of PII data